images/linux/scripts/helpers/Common.Helpers.psm1

```diff
@@ -54,4 +54,16 @@ function Get-AndroidPackages {
     $androidSDKManagerPath = "/usr/local/lib/android/sdk/tools/bin/sdkmanager"
     $androidPackages = & $androidSDKManagerPath --list --verbose
     return $androidPackages
+}
+
+function Get-EnvironmentVariable($variable) {
+    return [System.Environment]::GetEnvironmentVariable($variable)
+}
+
+function Update-Environment {
+    $variables = Get-Content "/etc/environment" | Select-String -NotMatch "^#"
+    $variables | ForEach-Object { 
+        $variable = $_[0].ToString().split("=")
```

Comment:
I think you should use `Split("=", 2)` here.
Real example, some time ago we had variable `JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8` in `/etc/environment`. Your code will break this variable

Reply:
thank you, I'll fix it ;)